.gitlab/build-deb-rpm.sh

```diff
@@ -5,7 +5,7 @@ echo -n "$RUBY_PACKAGE_VERSION" > auto_inject-ruby.version
 
 source common_build_functions.sh
 
-chmod a+r -R ../tmp/*
+chmod -R a+r,go-w ../tmp/*
```

Comment:
Can you add a comment here explaining what the permission change is accomplishing, so we can reference it in the future?

Reply:
Is `../tmp` here the same as the top-level `/tmp` in the root of the filesystem?

If so, should we maybe use a subdirectory of `tmp`? E.g. so that we don't accidentally package other things that may have left files in `/tmp`?

Reply:
> Is ../tmp here the same as the top-level /tmp in the root of the filesystem?

No, this is under the `dd-trace-rb`